gma/src/include/firebase/gma/types.h

```diff
@@ -216,16 +216,15 @@ class AdError {
   /// converting error information from legacy mediation adapter callbacks.
   static const char* const kUndefinedDomain;
 
- protected:
-  /// Constructor used when building results in Ad event callbacks.
-  explicit AdError(const AdErrorInternal& ad_error_internal);
```

Comment:
If this needs to stay protected for some reason, you can always surround it by #if !defined(DOXYGEN) to hide it.